MPF.Processors/DiscImageCreator.cs

```diff
@@ -820,6 +821,33 @@ internal override List<OutputFile> GetOutputFiles(string? baseDirectory, string
 
         #region Information Extraction Methods
 
+        /// <summary>
+        /// Get the dumping parameters from the log, if possible
+        /// </summary>
+        /// <param name="dicCmd">Log file location</param>
+        /// <returns>Dumping parameters if possible, null otherwise</returns>
+        public static string? GetParameters(string? dicCmd)
+        {
+            // If the file doesn't exist, we can't get the info
+            if (string.IsNullOrEmpty(dicCmd))
+                return null;
+            if (!File.Exists(dicCmd))
+                return null;
+
+            try
+            {
+                // Dumping parameters are on the first line
+                using var sr = File.OpenText(dicCmd);
+                var line = sr.ReadLine();
```

Comment:
Is it worth reading the whole file and not just the first line? I know it should only be one line, but still.

Reply:
I restricted to one line just in case of some weird file with 100+ lines